test/scripts/jenkins_visual_regression.sh

```diff
@@ -27,7 +27,7 @@ fi
 export TEST_BROWSER_HEADLESS=1
 
 checks-reporter-with-killswitch "Kibana visual regression tests" \
-  yarn run percy exec \
+  yarn run percy exec -t 500 \
```

Comment:
What does `-t` do?

Reply:
@spalger it is a network idle timeout flag.  Sometimes we are missing fonts captured in snapshots because of delays to Percy which does the asset discovery.  The default is 50 ms but from what I am seeing we need to bump this up.  I am using 500 ms in the external runs I am doing and did not see the icon problems and so far.  So let's give it a try, it should resolve the icon issue with maps and may resolve most, if not all the issues with the discover tests, however I don't have enough data yet on discover tests, but I continue to run externally to see if I can uncover any other issues.

Reply:
Okay, as long as we aren't planning on turning them back on in master before we have enough data.